setup.py

```diff
@@ -1,48 +1,49 @@
-# -------------------------------------------------------------------------------
+#-------------------------------------------------------------------------------
 # pyelftools: setup.py
 #
 # Setup/installation script.
 #
 # Eli Bendersky (eliben@gmail.com)
 # This code is in the public domain
-# -------------------------------------------------------------------------------
+#-------------------------------------------------------------------------------
 import os, sys
 from setuptools import setup
 
 
 try:
-    with open("README", "rt") as readme:
-        description = "\n" + readme.read()
+    with open('README', 'rt') as readme:
+        description = '\n' + readme.read()
 except IOError:
     # maybe running setup.py from some other dir
-    description = ""
+    description = ''
 
 
 setup(
     # metadata
-    name="pyelftools",
-    description="Library for analyzing ELF files and DWARF debugging information",
+    name='pyelftools',
+    description='Library for analyzing ELF files and DWARF debugging information',
     long_description=description,
-    license="Public domain",
-    version="0.30",
-    author="Eli Bendersky",
-    maintainer="Eli Bendersky",
-    author_email="eliben@gmail.com",
-    url="https://github.com/eliben/pyelftools",
-    platforms="Cross Platform",
-    classifiers=[
-        "Programming Language :: Python :: 3",
-    ],
+    license='Public domain',
+    version='0.30',
+    author='Eli Bendersky',
+    maintainer='Eli Bendersky',
+    author_email='eliben@gmail.com',
+    url='https://github.com/eliben/pyelftools',
+    platforms='Cross Platform',
+    classifiers = [
+        'Programming Language :: Python :: 3',
+        ],
+
     # All packages and sub-packages must be listed here
     packages=[
-        "elftools",
```

Comment:
Not sure what all these changes have to do with your proposed addition, can you elaborate?

Reply:
Are you looking at an older commit?
I had some formatting done which I have removed in the latest commit.